File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/domain/converter/TspToTbspConverter.java
package com.tarbadev.witchcraft.domain.converter;

public class TspToTbspConverter implements IConverter {
  private static final Double TSP_IN_ONE_TBSP = 3.0;

  @Override
  public Double convert(Double number) {
    return number / TSP_IN_ONE_TBSP;
  }
}

File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/converter/TspToTbspConverter.kt
package com.tarbadev.witchcraft.converter

class TspToTbspConverter : IConverter {

    override fun convert(number: Double): Double {
        return number / TSP_IN_ONE_TBSP
    }

    companion object {
        private const val TSP_IN_ONE_TBSP = 3.0
    }
}
